collect_app/src/main/java/org/odk/collect/android/receivers/NetworkReceiver.java

```diff
@@ -94,32 +119,17 @@ private boolean isFormAutoSendOptionEnabled(NetworkInfo currentNetworkInfo) {
     }
 
     /**
-     * @param isFormAutoSendOptionEnabled represents whether the auto-send option is enabled at the app level
+     * If the app-level autosend setting is enabled, send all filled forms that don't specify not to
+     * autosend at the form level. If the app-level autosend setting is disabled, send all filled
```

Comment:
Wouldn't `send all finalized forms` be better here? I think a form might be filled but not finalized. The same above.